perun-base/src/test/resources/test-schema.sql

```diff
@@ -1352,6 +1352,18 @@ create table vos_vos (
 								constraint vos_vos_memid_fk foreign key (member_vo_id) references vos(id)
 );
 
+-- ALLOWED_GROUPS_TO_HIERARCHICAL_VO - Groups allowed to be included to parent vo's groups
+create table allowed_groups_to_hierarchical_vo (
+							group_id integer not null,     --identifier of group
+							vo_id integer not null,        --identifier of parent vo
+							created_at timestamp default statement_timestamp() not null,
+							created_by varchar default user not null,
+							created_by_uid integer,
+							constraint alwd_grps_pk primary key (group_id, vo_id),
+							constraint alwd_grps_gid_fk foreign key (group_id) references groups(id) on delete cascade,
+							constraint alwd_grps_void_fk foreign key (vo_id) references vos(id)
```

Comment:
Is there a reason for having delete cascade only on the groups foreign key? (maybe poor performance since vos don't get removed as much?) Not a problem with the code, just my curiosity :) .